mbtiles/src/metadata.rs

```diff
@@ -136,6 +139,7 @@ impl Mbtiles {
                         tj.other.insert(name, Value::String(value));
                     }
                     "agg_tiles_hash" => agg_tiles_hash = Some(value),
+                    "scheme" => (), // ignore scheme because it defaults to xyz
```

Comment:
I see the issue - so basically we treat `schema` as a custom field, and because your mbtiles just happen to have it set, I copy it into the output as is... I see how this could be a problem.

We should not silently ignore it though, so perhaps something like this?  (haven't tested, might use some wordsmithing)

```suggestion
                    "scheme" => {
                        if value != "tms" {
                            warn!("File {file} has an unexpected metadata value schema='{value}'. Only 'tms' is supported. Ignoring.")
                        }
                    }
```